tests/integration/backup/test_v3_backup.py

```diff
@@ -176,7 +177,7 @@ def test_roundtrip_from_kafka_state(
             "--topic",
             new_topic.name,
             "--location",
-            str(metadata_path),
+            str(backup_directory),
```

Comment:
I think it's overkill to introduce a specific integration test for this feature, so I changed only this test to verify passing a parent directory works. The other ones passes the metadata file path.